Document `$replica()` method, remove not working `queryRaw` routing (#25)
prisma/extension-read-replicas: README.md

All non-transactional read queries will now be executed against the defined replica.  
Write queries and transactions will be executed against the primary server.

> **Note**: `queryRaw` and `executeRaw` are always executed against primary server by default since
> extension can not know for sure if particular raw query would read or write to the DB.
> Use `$replica()` method to explicitly route the request to read replica.

Reviewer: ```suggestion
**Note**: `queryRaw` and `executeRaw` are always executed against the primary server by default since
the extension can not know for sure if a particular raw query would read or write to the database.
Use the `$replica()` method to explicitly route the request to a read replica.
```

prisma.$primary().user.findMany({ where: { ... }})
```

> ### Forcing request to go through replica
> 
> Sometimes you might want to do the opposite and route the request to the replica even though
> it will be routed to primary by default. In that case, you can use `$replica()` method:
> 
> ```ts
> prisma.$replica().$queryRaw`SELECT ...`
> ```
> 
> If multiple replicas are configured, random one will be selected.

Reviewer: ```suggestion
### Forcing request to go through a replica

Sometimes you might want to do the opposite and route the request to a replica even though
it will be routed to primary by default. In that case, you can use the `$replica()` method:

```ts
prisma.$replica().$queryRaw`SELECT ...`
```

If multiple replicas are configured, one will be selected randomly.
```
